Makefile.BSD

```diff
@@ -14,7 +14,14 @@ ${D}/Makefile::
 	( cd ${D} && printf 'SRCS=' && ${MAKE} -V SRCS | sed -e 's| cpusupport-config.h||' ) >> $@
 	( cd ${D} && printf 'IDIRS=' && ${MAKE} -V IDIRS ) >> $@
 	( cd ${D} && printf 'LDADD_REQ=' && ${MAKE} -V LDADD_REQ ) >> $@
-	cat Makefile.prog >> $@
+	# Copy the Makefile template, removing the "install:"
```

Comment:
You know this is in `Makefile.BSD`, right?  So it's only getting run by BSD make on either your computer or mine?  :)

Still, updated PR accordingly.  Once this is accepted, I'll send in another PR that moves the other comments in this file outside of the rules.  (I don't want to do this now, because that would cause merge conflicts between the two PRs, so I'd have to edit it later anyway)